torchani/models.py

```diff
@@ -95,6 +95,21 @@ def __init__(self, info_file, periodic_table_index=False):
         self.neural_networks = neurochem.load_model_ensemble(
             self.species, self.ensemble_prefix, self.ensemble_size)
 
+    @classmethod
+    def from_pt(cls, pt_file, periodic_table_index=False, force_build=False):
```

Comment:
Can we make `from_pt` the default option when creating an instance of classes

Reply:
If it's not the default, it's very likely that no one will use it.

Reply:
I think it would be a bit difficult to make it the default because we would have to overwrite __new__ somehow, __init__ doesn't really return an instance, it just initializes the instance once it has already been built.